src/gridtools/CMakeLists.txt

```diff
@@ -14,7 +14,8 @@
 ##
 ##===------------------------------------------------------------------------------------------===##
 
+include(msbuildAddCustomDummyTarget)
 # Add a dummy target with all the source files so that they are included in a qt creator project
-dawn_add_custom_dummy_target(NAME qt_creator_parse_project_clang
+msbuild_add_custom_dummy_target(NAME qt_creator_parse_project_clang
```

Comment:
Might be worth sticking to a convention of naming targets (in this case something like `DawnQtCreatorDummyTarget` would be more fitting).

Reply:
sure, should it be `GTClangQtCreatorDummyTarget` instead?